package.json

```diff
@@ -98,6 +99,7 @@
     "react-sticky": "^6.0.3",
     "react-virtualized": "^9.19.1",
     "recharts": "^2.0.0-beta.1",
+    "styled-components": "^5.0.0",
```

Comment:
why?

Reply:
`styled-components` imports in `EXPERIMENTAL_Table` were breaking without this dependency on root, e.g., [Cell.tsx:2](https://github.com/vtex/styleguide/blob/master/react/components/EXPERIMENTAL_Table/DataTable/Cell.tsx#L2)

Reply:
wow, I didn't know we were using styled-components. This should not have happened.